packages/contracts/src/dollar/DirectGovernanceFarmer.sol

```diff
@@ -73,11 +73,11 @@ contract DirectGovernanceFarmer is ReentrancyGuard {
     //TODO create updateConfig method
     /// Needs to check that Operator is Authorized, From is Valid, ID exists
     function onERC1155Received(
-        address operator,
-        address from,
-        uint256 id,
-        uint256 value,
-        bytes calldata data
+        address /* operator */,
+        address /* from */,
+        uint256 /* id */,
+        uint256 /* value */,
+        bytes calldata /* data */
```

Comment:
Seems unusual to me that these are all commented out.

Reply:
these are not commented out. This pattern says to the contract to use the var only when available on a child contract

Reply:
also are the same changes than the last one! with only new push us to slightly changes due to new commits on development. but you may ask me anything @pavlovcik

Reply:
this is a comment in solidity 

`// Comment`

`/* var */` this is a pattern that preservers contracts pattern to be use or that are being use in child contract